### ISSUE TITLE
Pods jquery ui css messing with gravity forms

### ISSUE THREAD

**Issue by saltnpixels** [pods-framework/pods#4612]
Eric (shamai) [9:03 PM] 
https://www.dropbox.com/s/vpk354ghlpmkev3/Screenshot%202017-12-05%2021.02.54.png?dl=0
jquery ui of pods messing with gravity forms edit screen
![screenshot 2017-12-05 21 02 54_preview](https://user-images.githubusercontent.com/3052875/33640821-c9fd8256-da00-11e7-841d-7f9dc09fad94.png)

how it should look:
https://www.dropbox.com/s/ajvci2p9ljsnnhf/Screenshot%202017-12-05%2021.02.20.png?dl=0
![screenshot 2017-12-05 21 02 20_preview](https://user-images.githubusercontent.com/3052875/33640841-ea7675ce-da00-11e7-81d9-ad5687d109e5.png)

how it does look:
https://www.dropbox.com/s/6gu5jdxo0f6bffa/Screenshot%202017-12-05%2021.05.42.png?dl=0
![screenshot 2017-12-05 21 05 42_preview](https://user-images.githubusercontent.com/3052875/33640857-fb864182-da00-11e7-8a96-90f3f9a04756.png)

**Comment by sc0ttkclark** [comment]
@nicdford @pglewis

Looks like during the SCSS refactor we started pulling ALL of our CSS library code any time we need any of our CSS -- it's now one big file /ui/styles/src/pods.scss that gets built from.

Here's the exact diff line where this is made clear:

https://github.com/pods-framework/pods/pull/4455/files#diff-e34156692b93ef7d263924ac927338f2L238

That's PodsInit.php at line 238 in register_assets where lines like these are removed where we were registering stylesheets only if they weren't registered before (in case WP adds it's own jquery-ui -- a good check)

```php
if ( ! wp_style_is( 'jquery-ui', 'registered' ) ) {
    wp_register_style( 'jquery-ui', PODS_URL . 'ui/css/smoothness/jquery-ui.custom.css', array(), '1.8.16' );
}
```

That's the case for this and other libraries now. We no longer check if they exist already, we just load our CSS up and it starts to work.

One fix should be that we wrap our `@import` for the libs scss with a class like:

```css
form.pods-form, form.pods-admin {
	@import "libs/jquery-ui-custom";
	@import "libs/jquery-qtip";
}
```

Coincidentally, we need to do this same thing for our overrides too:

```css
form.pods-form, form.pods-admin {
	@import "overrides/codemirror";
	@import "overrides/jquery-cleditor";
	@import "overrides/jquery-ui-timepicker";
}
```
 
We also have problematic classes in other areas of the scss, just look at any cases where we're not using a `pods-` prefixed class like `.left` and others unless it's like `div.open a.pods-quick-add`, that's not too bad since it still only affects our elements.

I'm *still not sold* that our `jquery-ui` CSS should be loaded in this way, given that there's still some interest to have some sort of default jQuery UI CSS put into core (albeit not much of a fire under that task: https://core.trac.wordpress.org/ticket/18909).

**Comment by nicdford** [comment]
Would it be more valuable for us to reinstate the jquery-ui check in case its already loaded? My original thought process was to consolidate ALL of the CSS required to run Pods, but that has shown to not be a good idea. @pglewis and I already identified two cases (pods-form & pods-wizard) that it made more sense to isolate them because they were particularly large or weren't necessarily important for the majority of Pods. Should external dependencies also fall into this boat? 

As far as overrides go... in my opinion, it probably makes the most sense for those to stay in the main stylesheet with the `@import` method @sc0ttkclark mentioned above. 

What do you guys think? I'm happy to go either way, just want to make sure i'm on the same page. :)

**Comment by nicdford** [comment]
Specificity issues with related libraries fixed with #4632
